scripts/tests_like_ci/rust_tests.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
I would call the folder `ci_tests` and actually also update the workflow files to call these scripts, instead of doing the magic in the workflow files. At least that was the idea in the beginning.